docs/io/ascii/read.rst

```diff
@@ -577,6 +577,15 @@ The default converters for each column can be overridden with the
   ...               'col2': [ascii.convert_numpy(np.float32)]}
   >>> ascii.read('file.dat', converters=converters)  # doctest: +SKIP
 
+In addition to single column names you can use wildcards via `fnmatch` to
+select multiple columns. For example, we can set the format for all columns
+where the column name starts with "col" to `np.uint` while applying default
```

Comment:
Readthedocs fails, because it doesn't know about `np` - this should be `numpy.uint` (or maybe just "unsigned integer"?)